src/ui/organisms/PeoplePicker.test.tsx

```diff
@@ -59,4 +59,13 @@ describe('PeoplePicker', (): void => {
                 ),
         ).not.toThrow();
     });
+
+    it('should hide the external label when hideLabel is passed in', () => {
```

Comment:
assuming there is a +ve test for this to - i.e. it shows the label?

Reply:
There is now